src/mongosh/data/curriculum.json

```diff
@@ -1,6 +1,10 @@
 [
     {
-        "adhoc": true,
-        "name": "test"
+        "resources": [
+            {
+                "adhoc": true,
+                "name": "test"
+            }
+        ]
     }
-]
\ No newline at end of file
+]
```

Comment:
Let's build more test data - examine the Topics test data for Topic/Resource ID's to use, and People data for PersonID values to re-use as curriculumId's.